fs/config/config.go

```diff
@@ -68,3 +70,13 @@ type DirectoryCacheConfig struct {
 	SyncAdd          bool `toml:"sync_add"`
 	Direct           bool `toml:"direct"`
 }
+
+type FuseConfig struct {
+	// This defines overall timeout attribute for a file system in seconds.
+	//
+	AttrTimeout int64 `toml:"attr_timeout"`
+
+	// This defines TTL for directory, name lookup in seconds.
+	//
```

Comment:
Could you replace `this` to `EntryTimeout`? And the empty line isn't needed.

Reply:
Ack